asset/src/count_by_field/processor.ts

```diff
@@ -0,0 +1,43 @@
+import {
+    MapProcessor, DataEntity, WorkerContext, ExecutionConfig
+} from '@terascope/job-components';
+import { CountByFieldConfig } from './interfaces';
+import { JobMetricsAPI } from '../job_metric_api/interfaces';
+
+export default class CountByField extends MapProcessor<CountByFieldConfig> {
+    private metrics!: JobMetricsAPI;
+    private opName!: string;
+    static metrics: any;
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
since the constructor is just passing args to super you should be able to remove the constructor all together

Reply:
I removed the constructor.